src/Nri/Ui/ClickableText/V3.elm

```diff
@@ -394,6 +394,22 @@ appearsInline =
         ]
 
 
+{-| -}
+inMenu : Attribute msg
+inMenu =
+    set
+        (\config ->
+            { config
+                | rightIconStyles =
+                    List.append config.rightIconStyles
+                        [ Css.width (Css.px 15)
+                        , Css.height (Css.px 15)
+                        , Css.marginLeft (Css.px 8)
+                        ]
+            }
+        )
```

Comment:
I like the rightIconStyles idea!

What do you think about exposing `rightIconCss : List Css.Style -> Attribute msg` instead, and have the Menu component do the styling?

Reply:
I like it! Updated the code!